app/code/Magento/Authorization/Test/Integration/Console/Command/AdminRolesListCommandTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Authorization\Test\Integration\Console\Command;
+
+use Magento\Authorization\Command\AdminRolesListCommand;
+use Magento\TestFramework\Helper\Bootstrap;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Tester\CommandTester;
+
+/**
+ * Class AdminRolesListCommandTes
+ */
+class AdminRolesListCommandTest extends TestCase
+{
+    /**
+     * @magentoDataFixture ./../../../../app/code/Magento/Authorization/Test/Integration/_files/test_role.php
```

Comment:
I'm not sure, but you should be able to use
```suggestion
     * @magentoDataFixture Magento/Authorization/_files/test_role.php
```

Reply:
A tried it first, unfortunately this nice fixtures declaration works only inside dev/tests folder.

Reply:
OK